### PR TITLE
Homework seven v2

### PR DESCRIPTION
**Pull request by TimurIsroilov123**
realised downloading data from API such as movies, its details and actors. I remember that u also recommended to use pagination but because I lagged behind I'm planing catch up current lesson then come back to it )))

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (Axrorxoja) [inline .idea/codeStyles/Project.xml]: don't add this file to PR
add .idea folder to .gitignore file

File: .idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (Axrorxoja) [inline .idea/codeStyles/codeStyleConfig.xml]: previous moment

File: app/src/main/java/com/bignerdranch/android/androidacademy/data/MoviesLoad.kt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    suspend fun loadDetails(id: Long): MovieDetails {
+        return RetrofitModule.moviesApi.getMovieDetails(id)
+    }
+
+    suspend fun loadActors(id: Long): Credit {
+        return RetrofitModule.moviesApi.getActors(id)
+    }
+}
+
+private interface MoviesApi {
```
Comment (Axrorxoja) [inline app/src/main/java/com/bignerdranch/android/androidacademy/data/MoviesLoad.kt]: we agreed that to separate Movies api and retrofit module to another file

Comment (TimurIsroilov123) [reply app/src/main/java/com/bignerdranch/android/androidacademy/data/MoviesLoad.kt]: movies api moved to another file

File: .gitignore [hunk 4]
```diff
@@ -41,6 +41,8 @@ captures/
 .idea/workspace.xml
 .idea/tasks.xml
 .idea/gradle.xml
+.idea/codeStyles/Project.xml
+.idea/codeStyles/codeStyleConfig.xml
```
Comment (Axrorxoja) [inline .gitignore:45]: just .idea enough

**Review by Axrorxoja — APPROVED** [review]

File: .gitignore (hunk 4 above)
Comment (TimurIsroilov123) [reply .gitignore:45]: Got it

**Pull request merged**